### PR TITLE
Locker bloat removal

### PR DESCRIPTION
**Pull request by Kaffe-work**
## About The Pull Request

Removes a lot of useless stuff from Head of Staff lockers. 

## Why It's Good For The Game

#47203 was closed because there is already too much bloat in the lockers of the heads of staff; this removes unecessary stuff: zipties, gas masks and other stuff that doesn't really need to be there, or under-utilized tools in CE's case. I do not have enough experience with CMO to know what can be removed there. 

One clear positive from this change is lower lag from looking at head of staff locker contents. 

However, i am not in complete agreement with these changes - Perhaps a discussion of what the purpose of department heads actually is should be done: I think player experience and maintainer vision differs quite a bit here.

## Changelog
:cl: IradT
del: Removed a lot of stuff from Head of department lockers. 
/:cl:

### CODE REVIEW

**Comment by dr-bee** [comment]
Advanced extinguisher needs to stay in the CEs locker as it is one of three on the station and is used to put out SM fires.

**Comment by TheChosenEvilOne** [comment]
Please do not remove the advanced fire extinguisher from CE's locker.

**Comment by Kaffe-work** [comment]
re-added that one, mixed it up with the compact one.

**Comment by Shadowflame909** [comment]
Might I recommend leaving the inducer?

until science researches them. They're actually pretty limited

**Comment by Kaffe-work** [comment]
Honestly, I'd leave fucking everything - im just removing things that i think is less important than a RCD. Oranges closed the PR that added one to the CE locker, because of "bloat" so I'm removing "bloat"

**Comment by carlarctg** [comment]
i feel like there should be two separate lockers, one for clothing and another for actual equipment

**Comment by AdmiralPancakes1** [comment]
>I do not have enough experience with CMO to know what can be removed there.

The duffel bag and the medical belt can be removed, you can get these out of medbay's vending machines.

**Comment by Tlaltecuhtli** [comment]
remove the skirts

**Comment by Ohagi-Chan** [comment]
Spare head uniforms could probably be handled by the HoP.

**Comment by optimumtact** [comment]
good change

**Comment by optimumtact** [comment]
also you're still not going to get your RCD in anyway

File: code/game/objects/structures/crates_lockers/closets/secure/scientist.dm [hunk 1]
```diff
@@ -20,7 +20,6 @@
 	new /obj/item/clothing/gloves/color/latex(src)
 	new /obj/item/radio/headset/heads/rd(src)
 	new /obj/item/tank/internals/oxygen(src)
```
Comment (Fikou) [inline code/game/objects/structures/crates_lockers/closets/secure/scientist.dm]: this seems like bloat

File: code/game/objects/structures/crates_lockers/closets/secure/scientist.dm [hunk 2]
```diff
@@ -20,7 +20,6 @@
 	new /obj/item/clothing/gloves/color/latex(src)
```
Comment (Fikou) [inline code/game/objects/structures/crates_lockers/closets/secure/scientist.dm]: this too

File: code/game/objects/structures/crates_lockers/closets/secure/engineering.dm [hunk 3]
```diff
@@ -10,25 +10,18 @@
...
 	new /obj/item/areaeditor/blueprints(src)
-	new /obj/item/airlock_painter(src)
 	new /obj/item/holosign_creator/engineering(src)
-	new /obj/item/clothing/mask/gas(src)
-	new /obj/item/multitool(src)
 	new /obj/item/assembly/flash/handheld(src)
 	new /obj/item/clothing/glasses/meson/engine(src)
 	new /obj/item/door_remote/chief_engineer(src)
 	new /obj/item/pipe_dispenser(src)
-	new /obj/item/inducer(src)
```
Comment (Fikou) [inline code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]: why

**Comment by Fikou** [comment]
the latex gloves and oxygen tank from rd's locker seem like bloat too

**Comment by PepperPrepper** [comment]
How about most of the bloat is moved into a head of staff gear vendor that's located at the bridge?

**Comment by Detective-Google** [comment]
HoP should keep his zipties removing them would only 1. further enable people to bumrush newbie hops and steal aa and 2. make the hop even more likely to go raid security for gamer gear

**Comment by YakumoChen** [comment]
Remove the photo albums and put them nearby

**Comment by Kaffe-work** [comment]
removed CMO+RD stuff

**Comment by anconfuzedrock** [comment]
skirts do unironically cause lockers to overfloweth

**Comment by optimumtact** [comment]
skirts really need to be a preference

**Comment by actioninja** [comment]
They are a preference

**Comment by optimumtact** [comment]
then why are they in lockers?

**Comment by PepperPrepper** [comment]
From memory skirts are a preference yet they were also included in the heads lockers for whatever reason

**Comment by Kaffe-work** [comment]
> then why are they in lockers?

probably same reason as regular suits + other clothing garbage exists in head lockers: to suit up in case you are promoted to the job. 

This could really be solved by a vendor in bridge for heads instead, lets be real.

**Comment by Jalleo** [comment]
We can add it as a vendor to the bridge meeting room. Heads of staff uniform vendor.

File: code/game/objects/structures/crates_lockers/closets/secure/engineering.dm [hunk 4]
```diff
@@ -10,25 +10,18 @@
...
 	new /obj/item/circuitboard/machine/techfab/department/engineering(src)
 	new /obj/item/extinguisher/advanced(src)
 	new /obj/item/storage/photo_album/CE(src)
```
Comment (Jalleo) [inline code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]: Does anyone use this?

Comment (Bombpants) [reply code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]: The hazard vest keeps the CE Space-OSHA compliant

File: code/game/objects/structures/crates_lockers/closets/secure/medical.dm [hunk 5]
```diff
@@ -64,7 +63,6 @@
 	new /obj/item/megaphone/command(src)
 	new /obj/item/defibrillator/compact/loaded(src)
 	new /obj/item/clothing/gloves/color/latex/nitrile(src)
-	new /obj/item/storage/belt/medical(src)
```
Comment (actioninja) [inline code/game/objects/structures/crates_lockers/closets/secure/medical.dm]: I use both duffel and medbelt nearly every round as CMO.
They're available elsewhere, but these aren't just random items that sit unused.

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 6]
```diff
@@ -57,7 +56,6 @@
 	new /obj/item/clothing/suit/armor/vest/alt(src)
 	new /obj/item/assembly/flash/handheld(src)
 	new /obj/item/clothing/glasses/sunglasses(src)
-	new /obj/item/restraints/handcuffs/cable/zipties(src)
```
Comment (actioninja) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: ties in hoplocker are very useful, hop gets tided a shitload.
In a way it encourages hopcurity, but dealing with shitheads that run in or climb over the desk is a pain.

**Review by actioninja — COMMENTED** [review]
This isn't a formal review, but just some thoughts about the changes

**Comment by actioninja** [comment]
thinking more about it bitching because it would personally affect me is exactly what is most obnoxious here.

File: code/game/objects/structures/crates_lockers/closets/secure/medical.dm (hunk 5 above)
Comment (Kaffe-work) [reply code/game/objects/structures/crates_lockers/closets/secure/medical.dm]: it's easily dispensable from the medivend, much like the RCD is dispensable from engivend : angry:

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm (hunk 6 above)
Comment (Kaffe-work) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: there are handcuffs in bridge

**Pull request merged**